src/ui/App.tsx

```diff
@@ -17,45 +17,81 @@ import { IncomingRequest } from "./pages/IncomingRequest";
 
 setupIonicReact();
 
+const VERCEL_URL = "https://cf-identity-wallet.vercel.app";
+
 const App = () => {
   const dispatch = useAppDispatch();
   const currentOperation = useAppSelector(getCurrentOperation);
   const toastMsg = useAppSelector(getToastMsg);
   const [showScan, setShowScan] = useState(false);
   const [showToast, setShowToast] = useState(false);
 
+  const shouldLoadIframe = () => {
+    const isMobileParamPresent = new URLSearchParams(
+      window.location.search
+    ).has("mobile");
+    const isIframeLoaded = localStorage.getItem("isIframeLoaded") === "true";
+    const isNotInIframe = window.self === window.top;
+
+    return (isNotInIframe && !isIframeLoaded) || isMobileParamPresent;
```

Comment:
Since the iframe URL does _not_ contain the `mobile` query param, is it not just enough to just `return `isMobileParamPresent`?

Outer becomes iframe, and within iframe it's normal

Reply:
@iFergal iFrame was a bad idea that came out of that meeting we had earlier and I have now removed it.